DESCRIPTION

```diff
@@ -1,14 +1,42 @@
 Package: pacta.multi.loanbook.plot
-Title: What the Package Does (One Line, Title Case)
+Title: Tools to Visualize Climate Metrics for Multiple Loanbooks
 Version: 0.0.0.9000
 Authors@R: 
-    person("First", "Last", , "first.last@example.com", role = c("aut", "cre"),
-           comment = c(ORCID = "YOUR-ORCID-ID"))
-Description: What the package does (one paragraph).
+    c(person(given = "Jacob",
+             family = "Kastl",
+             role = c("aut", "cre"),
+             email = "jacob.kastl@gmail.com"),
+      person(given = "Monika",
+             family = "Furdyna",
+             role = c("aut", "ctb"),
+             email = "monika.furdyna@gmail.com",
+             comment = c(ORCID = "0000-0002-3728-0646")),
```

Comment:
maybe we default to @MonikaFu being maintainer here right away? or what do you two think?

Reply:
Fine with me, if @MonikaFu does not disagree. But we can change it back if she does

Reply:
Definitely, not trying to sign her up for something she doesn't want! (I'm just 99% sure she wants to maintain this 😜) 

Anyway not a big deal we can make an issue and change it when she's back from PTO